eyre/src/lib.rs

```diff
@@ -265,7 +265,17 @@
 //! vice-versa by re-exporting all of the renamed APIs with the names used in
 //! `anyhow`, though there are some differences still.
 //!
-//! #### `Context` and `Option`
+//! ### Disabling the compatibility layer
```

Comment:
Great.

This further goes into the general guideline that library authors should default to `default_features = false`